HW03.rmd

```diff
@@ -0,0 +1,56 @@
+---
+title: "HW03"
+author: "Ben Wang"
+date: "7/20/20"
+output: github_document
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+library(ggplot2)
+library(tidyverse)
+library(dplyr)
+```
+
+### I decided to use the NYTimes coronavirus database to plot positive coronavirus cases in Illinois and New York, because these are the two places my brother and I are living. Below is the code for pulling the data from the New York Times database
+
+```{r}
+states <- read.csv(url("https://raw.githubusercontent.com/nytimes/covid-19-data/master/us-states.csv"))
```

Comment:
This is exactly what I was looking to do but could not figure out how to code it!  I ended up downloading the data to my computer and then dragging it into my project folder but this is way better.

Reply:
I was doing the same, and was also considering cloning the whole folder but I'm not sure if that works. This is exactly what I'm looking for lol

Reply:
Woah, I didn't know that we can make R to read csv files from a link directly! Do you know if the same technique can be used for other types of files such as tsv and excel?

Reply:
Woah, I didn't know that we can make R to read csv files from a link directly! Do you know if the same technique can be used for other types of files such as tsv and excel?